docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -184,7 +184,7 @@ In the .NET Framework 4.7.2, WPF added the ability to draw <xref:System.Windows.
 
 To use this property, simply add it to your XAML code and use the appropriate brush or binding. The resulting text selection looks like this:
 
-![Tooltip when user navigates to a button with the keyboard](./media/selectiontextbrush-property.png)
+![Screenshot of what it looks like when you use the SelectionTextBrush.](./media/whats-new-in-accessibility/selectiontextbrush-property.png)
```

Comment:
This alt text doesn't help someone who can't see the image.

It could be something like "Screenshot of the app running with the word Hello World selected."

A similar exercise should be done throughout the rest of this article.